cs/components.texy

```diff
@@ -34,14 +34,13 @@ Bavíme-li se o komponentách, obvykle myslíme potomky třídy [Control |api:Ne
 Komponenta obsahuje továrnu na svou [šablonu |latte:]. Ta standardně vytvoří šablonu, předá ji některé základní proměnné a zaregistruje [standardní filtry |latte:filters]. O vykreslení se už musíme postarat sami, a to v metodě `render()`. Tam také musíme určit soubor, ze kterého bude šablona načtena, a zaregistrovat proměnné, které se budou v šabloně používat. Šablonu můžeme umístit do stejné složky a pod stejným názvem jako komponentu:
 
 /--php
-public function render()
+public function render(): void
 {
-	$template = $this->template;
-	$template->setFile(__DIR__ . '/poll.latte');
+	$this->template->setFile(__DIR__ . '/poll.latte');
 	// vložíme do šablony nějaké parametry
-	$template->param = $value;
+	$this->template->param = $value;
 	// a vykreslíme ji
-	$template->render();
+	$this->template->render();
 }
```

Comment:
Možná by tu bylo lepší rovnou `$this->template->render(__DIR__ . '/poll.latte');`

Reply:
Rozumim.

Reply:
Done.